docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -9,7 +9,11 @@ The TestCafe API includes methods that open, close, and switch between browser w
 
 ⚠ This is a **beta** feature. Browser support is limited to local instances of Chrome and Firefox. You cannot resize child windows. You cannot take screenshots or videos of child windows. The available functionality is subject to further revisions. Please do not use this feature in production environments.
 
-️🛠️ Use the `--disable-multiple-windows` CLI flag to disable support for multiple browser windows if you encounter compatibility issues with your tests.
+️🛠️ You can disable support for multiple browser windows if you encounter compatibility issues with your tests. Use either of the following settings:
+
+* the [--disable-multiple-windows](../../reference/command-line-interface.md#--disable-multiple-windows) command line flag,
+* the `disableMultipleWindows` option in the [runner.run](../../reference/testcafe-api/runner/run.md) method,
```

Comment:
```suggestion
* the `disableMultipleWindows` [runner.run](../../reference/testcafe-api/runner/run.md) option,
```
Options are *passed to* the method during the call, therefore the method itself can not _contain_ any options (right?). I can't think of a way to eloquently express this, so I just removed any references to the nature of these entities' relationship 😆 Feel free to leave the sentence as it was, though, it might just be a matter of taste

Reply:
I like your version better, awesome, thanks, though I didn't mean that the method *contains* its options. Do you feel like *in* necessarily implies containing in this context?